internal/volume/volumes_information.go

```diff
@@ -92,6 +92,9 @@ type BackupVolumeInfo struct {
 	// Snapshot completes timestamp.
 	CompletionTimestamp *metav1.Time `json:"completionTimestamp,omitempty"`
 
+	// Whether the volume data is backed up successfully.
+	Succeeded bool `json:"succeeded,omitempty"`
```

Comment:
For long term consideration, there are at lease 3 terminating states --- succeeded, failed, cancelled. So could we consider to make it as a string?

Reply:
Modified.